### PR TITLE
Finished Acts 8

### PR DESCRIPTION
**Pull request by EvanZhouDev**
Finished animations for Acts 8.

Known Issues:
- Formatting doesn't work on some browsers (not fixed, will do later, not the most important right now)

### CODE REVIEW

**Comment by DragonXDev** [comment]
Few things:
1. Some spacing is off with "Peter" and "Saul's Journey"
2. Few issues while scrolling in the drawer; at one point u can scroll and u have to move your mouse to the right side to continue
3. Saul randomly disappears sometimes and doesn't show while scrolling
4. Transitions can be added for the background color change
5. Possibly add an animation of Saul preaching to a group of people

File: public/assets/styles/acts8_9.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    width: 25%;
+    right: 5%;
+}
+.words5 {
+    position: absolute;
+    top: 200%;
+    width: 25%;
+    right: 5%;
+}
+.words6 {
```
Comment (DragonXDev) [inline public/assets/styles/acts8_9.css:32]: naming conventions...

**Review by ThatXliner — CHANGES_REQUESTED** [review]
I’m not done but here we have so far

**Comment by ThatXliner** [comment]
Screen Shot 2022-10-30 at 5 49 46 PM
screwy....

**Comment by ThatXliner** [comment]
There could also be a transition from black back to white

File: index.html [hunk 2]
```diff
@@ -13,7 +13,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.11.2/MotionPathPlugin.min.js"></script>
     <script type="module">
       window.$store = {
-        currentPage: 0,
+        currentPage: 3,
```
[the remark this answers is not in the document]
Comment (EvanZhouDev) [reply index.html]: Fixed!

**Comment by EvanZhouDev** [comment]
Many of the things you have requested can be resolved with percentages. I will try to implement that.

**Pull request merged**